redis/asyncio/cluster.py

```diff
@@ -1450,8 +1462,9 @@ async def _execute(
                     )
             if len(target_nodes) > 1:
                 raise RedisClusterException(f"Too many targets for command {cmd.args}")
-
             node = target_nodes[0]
+            if node == client.get_default_node():
+                is_default_node = True
```

Comment:
This is inside a for loop, so you might want an or operation here.

Reply:
Hey, please see the changes in the new commit.